src/NuGet/Microsoft.Net.Compilers.Toolset/build/Microsoft.Net.Compilers.Toolset.props

```diff
@@ -0,0 +1,17 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project DefaultTargets="Build">
```

Comment:
>DefaultTargets="Build" [](start = 9, length = 22)

I'm not sure if you should be specifying this. It is redundant with [https://github.com/Microsoft/msbuild/blob/99e150416ed43660782db8ea88b2d12135aca855/src/Tasks/Microsoft.Common.targets#L17](https://github.com/Microsoft/msbuild/blob/99e150416ed43660782db8ea88b2d12135aca855/src/Tasks/Microsoft.Common.targets#L17)

Reply:
Okay. It's another thing I grabbed from the template on the web tutorial. Will delete.